packages/ui/src/1_atoms/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useMemo, useCallback } from 'react';
+
+import classNames from 'classnames';
+
+import { TabColor, TabSize, TabStyle } from './Tabs.types';
+import { Tab } from './components/Tab/Tab';
+
+interface ITabItem {
+  label: React.ReactNode;
+  content: React.ReactNode;
+  disabled?: boolean;
+  dataActionId?: string;
+  activeColor?: TabColor;
+}
+
+type TabsProps = {
+  className?: string;
+  contentClassName?: string;
+  index: number;
+  items: ITabItem[];
+  onChange?: (index: number) => void;
+  style?: TabStyle;
```

Comment:
```suggestion
  type?: TabType;
```
According to the spec, this should be called `type`.